examples/pytorch/rgcn/model.py

```diff
@@ -1,166 +1,105 @@
+from dgl import DGLGraph
 import torch as th
 import torch.nn as nn
-
+import torch.nn.functional as F
 import dgl
 
-class BaseRGCN(nn.Module):
-    def __init__(self, num_nodes, h_dim, out_dim, num_rels, num_bases,
-                 num_hidden_layers=1, dropout=0,
-                 use_self_loop=False, use_cuda=False):
-        super(BaseRGCN, self).__init__()
-        self.num_nodes = num_nodes
-        self.h_dim = h_dim
-        self.out_dim = out_dim
-        self.num_rels = num_rels
-        self.num_bases = None if num_bases < 0 else num_bases
-        self.num_hidden_layers = num_hidden_layers
-        self.dropout = dropout
-        self.use_self_loop = use_self_loop
-        self.use_cuda = use_cuda
+from dgl.nn.pytorch import RelGraphConv
 
-        # create rgcn layers
-        self.build_model()
+class RGCN(nn.Module):
+    """
+    Parameters
+    ----------
+    in_dim : int
+        Input feature size or number of nodes
```

Comment:
It's ok if you don't want to have very long docstring in examples as long as the argument name is self-explaining, but don't leave something half done.

Reply:
Removed.